contracts/PreseasonTwo/BuilderNFTPreSeason02Implementation.sol

```diff
@@ -423,6 +446,12 @@ contract BuilderNFTPreSeason02Implementation is
     function getBuilderIdForToken(
         uint256 tokenId
     ) external view returns (string memory) {
```

Comment:
For simplicity, we can change statemutability to 

function getBuilderIdForToken(
        uint256 tokenId
    ) **public view** returns (string memory)
    
    
    This enables us to consume this method in external or internal functions